css/styles.css

```diff
@@ -0,0 +1,19 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+h1 {
+  text-align: center;
+  background-color: lightblue;
+}
+
+.overall-question {
+  margin: 0px;
+  padding: 10px 10px 10px 10px;
```

Comment:
if you want the same padding at all sides of your content, you can just type `padding: 10px;`